src/AspNet.Security.OAuth.Mixcloud/AspNet.Security.OAuth.Mixcloud.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>ASP.NET Core security middleware enabling Mixcloud authentication.</Description>
+    <Authors>Adam Reisinger;Chino Chang;Michael DeMond</Authors>
```

Comment:
No need to sign my name here, bro.

Reply:
Haha good point...  I copied/pasted your code straight up, so felt the need to attribute accordingly.  I will remove.